python/lsst/ts/scheduler/driver.py

```diff
@@ -10,6 +13,80 @@
 __all__ = ["Driver", "DriverParameters"]
 
 
+class DriverTarget(Target):
+    def __init__(self, targetid=0, fieldid=0, band_filter="",
+                 ra_rad=0.0, dec_rad=0.0, ang_rad=0.0,
+                 num_exp=0, exp_times=[]):
+        """Initialize the class.
+
+        Parameters
+        ----------
+        targetid : int
+            A unique identifier for the given target.
+        fieldid : int
+            The ID of the associated OpSim field for the target.
+        band_filter : str
+            The single character name of the associated band filter.
+        ra_rad : float
+            The right ascension (radians) of the target.
+        dec_rad : float
+            The declination (radians) of the target.
+        ang_rad : float
+            The sky angle (radians) of the target.
```

Comment:
I'm not sure your performance requirements, but... consider using astropy quantities for angles and possibly durations (e.g. `exp_times`) as well